qiime2/metadata.py

```diff
@@ -91,6 +102,69 @@ def load(cls, path):
             raise ValueError(invalid_metadata_template % msg)
         return cls(df)
 
+    def merge(self, *others):
+        """Merge this ``Metadata`` object with other ``Metadata`` objects.
+
+        Returns a new ``Metadata`` object containing the merged contents of
+        this ``Metadata`` object and `others`. The merge is not in-place and
+        will always return a **new** merged ``Metadata`` object.
+
+        The merge will include only those IDs that are shared across **all**
+        ``Metadata`` objects being merged (i.e. the merge is an *inner join*).
+
+        Each column being merged must be unique; merging metadata with
```

Comment:
I am anchoring this question here, but it might be applicable elsewhere. Maybe I am just being pedantic, but would it make more sense to refer to `MetadataCategories` in this docstring, instead of `column`s? Seems like it might be more consistent (and self-explanatory) to continue to refer to these as categories (maybe with an example illustrating that `MetadataCategories` are just a `column` in a table).